jaxopt/_src/anderson.py

```diff
@@ -108,6 +108,8 @@ class AndersonState(NamedTuple):
   residual_gram: jnp.array
   aux: Any
 
+  num_fun_eval: jnp.array = jnp.array(0, base.N_EVAL_DTYPE)
```

Comment:
Sorry, slight preference for `NUM_EVAL_DTYPE` for consistency

Reply:
sure, fixing rn